src/db/types.ts

```diff
@@ -7,7 +7,8 @@ export interface BaseModel {
 export type DBRoot = BaseModel;
 
 export interface DBGuild extends BaseModel {
-  readonly randomVoiceChannelNamesFrequency?: string;
+  readonly randomVoiceChannelNamesCron?: string;
```

Comment:
If we keep this I'll handle the db migration for this. Or I can just leave it alone. Not sure which is clearer

Reply:
I think since we don't yet have a way to run migration scripts, this should be left as it was before (since frequency more-or-less still accurately describes the field). Our next devops-y priority should be migration scripts and management of that, I would be shocked if there wasn't an open source tool for that kind of thing already.